### ISSUE TITLE
TimeoutError: waiting for selector 

### ISSUE THREAD

**Issue by rafaello9** [jibon57/bbb-recorder#46]
I launched the "Start Record" button of the webinar and then entered in the terminal

`node liveJoin.js "http://192.168.1.100/bigbluebutton/api/join?meetingId=random-8051339" liveRecord.webm 0 true`

Got an error

`{TimeoutError: waiting for selector "[aria-label =" Listen only "]" failed: timeout 30000ms exceeded`

**Comment by jibon57** [comment]
Please do search before post. This has been answered too many times.
